cookbooks/bcpc-hadoop/recipes/historyserver.rb

```diff
@@ -1,14 +1,20 @@
 include_recipe 'bcpc-hadoop::hadoop_config'
+::Chef::Recipe.send(:include, Bcpc_Hadoop::Helper)
+Chef::Resource::Bash.send(:include, Bcpc_Hadoop::Helper)
 
 %w{hadoop-mapreduce-historyserver}.each do |pkg|
-  package pkg do
+  package hwx_pkg_str(pkg, node[:bcpc][:hadoop][:distribution][:release]) do
     action :upgrade
```

Comment:
Other recipes did `install` instead of `upgrade`. Is there a standard. I believe `upgrade` was what we always did before.